### PR TITLE
Added a note about built-in priority-classes

### PR DESCRIPTION
**Pull request by alexzimmer96**
This PR adds a note about Kubernetes build-in PriorityClasses as requested in #15908.

### CODE REVIEW

**Comment by alexzimmer96** [comment]
I just signed the CLA.

File: content/en/docs/concepts/configuration/pod-priority-preemption.md [hunk 1]
```diff
@@ -62,6 +62,12 @@ To use priority and preemption in Kubernetes 1.11 and later, follow these steps:
 
 Keep reading for more information about these steps.
 
+{{< note >}}
+Kubernetes already ships with two PriorityClasses:
+`system-cluster-critical` and `system-node-critical`.
+These are common classes and are used to [ensure that critical components are always scheduled first](docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/).
```
Comment (sftim) [inline content/en/docs/concepts/configuration/pod-priority-preemption.md]: nit:
```suggestion
These are common classes and are used to [ensure that critical components are always scheduled first](/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/).
```

**Comment by sftim** [comment]
Hi @alexzimmer96 and thanks for opening your first PR. If you're happy to accept the suggestion I've made, I think this will be good to merge.

I'm going to tweak the PR title to fix a typo.
/retitle Added a note about built-in priority-classes

**Comment by alexzimmer96** [comment]
Hi @sftim 
I accepted your changes. Thanks for fixing my typo :)

**Pull request merged**